mapit_gb/management/commands/mapit_UK_show_missing_codes.py

```diff
@@ -30,6 +46,8 @@ def handle_noargs(self, **options):
                 type=area_type
             )
             for code_type in used_code_types:
+                continue if self.is_code_type_not_required_for_area_type(code_type, area_type)
```

Comment:
this needs to be `if self.is_code_type_not_required_for_area_type(code_type, area_type): continue` (an `else` clause is required when using an `if` as a statement modifier like in ruby - makes me think it's not really a full statement modifier).  As far as I can tell it looks like guard clauses are written in the `if condition: break` style for Python.